spec/formatting.md

```diff
@@ -93,17 +93,32 @@ and different implementations MAY choose to perform different levels of resoluti
 
 Depending on the presence or absence of an _operand_
 and a _function_, _private-use_, or _reserved_ _annotation_,
-one of the following is used to resolve the value of the _expression_:
-
-- If the _expression_ contains no _annotation_,
-  its resolved value is determined by _literal resolution_ or _variable resolution_,
-  depending on the shape of the _operand_.
-- Else, if the _expression_ has a _function_ _annotation_,
-  its resolved value is defined by _function resolution_.
-- Else, if the _expression_ has a _private-use_ _annotation_,
-  its resolved value is defined according to the implementations's specification.
-- Else, the _expression_ has a _reserved_ _annotation_,
-  an Unsupported Expression error is emitted and a fallback value is used as its value.
+the resolved value of the _expression_ is determined as follows:
+
+If the _expression_ contains a _reserved_ _annotation_,
+an `Unsupported Expression` error is emitted and a fallback value is used as its value;
+
+Else, if the _expression_ contains a _private-use_ _annotation_,
+its resolved value is defined according to the implementation's specification;
+
+Else, if the _expression_ contains an _annotation_,
+its resolved value is defined by _function resolution_;
+
+Else, the _expression_ will contain only an _operand_,
+which consists of either a _literal_ or a _variable_.
+
+If the _expression_ consists of a _variable_,
+its resolved value is defined by _variable resolution_.
+An implementation MAY use the value of the _variable_ or
+its resolved value to choose a _function_ and _options_
+to further perform _function resolution_.
+
+Else, if the _expression_ consists of a _literal_,
+its resolved value is defined by _literal resolution_.
+If the _literal_ appears in a _declaration_, no further resolution is permitted.
+Otherwise, an implementation MAY use the contents of the _literal_
+to choose a _function_ and _options_ to attempt to perform _function resolution_.
+If such a _function resolution_ fails, the _literal_ value MUST be the resolved value.
```

Comment:
```suggestion
```

I don't think this reflects the consensus achieved during the call; rather the opposite.
https://github.com/unicode-org/message-format-wg/blob/a53a0619a7dfcb0ce5cec5b1eb99e323d7dbbc85/meetings/2023/notes-2023-08-28.md?plain=1#L259-L263

Reply:
Done, but with a note added.